kong/plugins/request-transformer/access.lua

```diff
@@ -273,15 +321,15 @@ local function transform_multipart_body(conf, body, content_length, content_type
     end
   end
 
-  if removed or replaced or added or appended then 
+  if renamed or removed or replaced or added or appended then
     return true, parameters:tostring()
```

Comment:
(2/2) this call fails because `tostring()` tries to concatenate the previously given `value`, except that it is a table.